### DISCUSSION TITLE
Recommended way to display form submission button

### DISCUSSION THREAD

**Discussion by JasonTheAdams** [chakra-ui/chakra-ui#6053] [Q&A] [answered]
Greetings!

Been digging into Chakra UI for a project, and I've run into the fact that there doesn't seem to be a form button component. There are form components and there is a `Button` component, but it doesn't work when I set `as="input" type="submit"`, as is semantic for a form submission button. It also forces children, and inputs cannot have children. This says to me that the `Button` component is presently insufficient for this.

But as I can't possible be the first person to want a submission button for a form, I thought I'd reach out first and see what others are doing.

Thank you!

**Comment by chr-ge** [comment, marked as answer]
You can do `<Button type="submit">Submit</Button>`, no need to pass `as='input'`.

**Reply by JasonTheAdams** [reply]
I don't think that's right. This yields the following:
```jsx
<button type="submit" class="chakra-button css-110h3wq">Update Profile</button>
```
That is not semantically the same as:
```jsx
<input type="submit" value="Update Profile" />
```
One is simply a button which doesn't mean anything to the form. The latter is an input which triggers the submit event of the form (equal to pressing "enter" from a field).

**Reply by JasonTheAdams** [reply]
Turns out I'm wrong! The syntax you're describing _is_ semantically equivalent! I learned something new! Thanks! 👍
